scenes/menus/storybook/components/storybook.gd

```diff
@@ -68,24 +79,101 @@ func _ready() -> void:
 
 		previous_button = button
 
-	previous_button.focus_neighbor_bottom = back_button.get_path()
-	back_button.focus_neighbor_top = previous_button.get_path()
+	if previous_button:
+		previous_button.focus_neighbor_bottom = back_button.get_path()
+		back_button.focus_neighbor_top = previous_button.get_path()
 
+	left_button.pressed.connect(_on_left_button_pressed)
+	right_button.pressed.connect(_on_right_button_pressed)
 	reset_focus()
 
 
+func _change_focus_to(index: int) -> void:
+	if _navigation_locked:
+		return
+
+	if not quest_list:
+		return
+	var total := quest_list.get_child_count()
+	if total == 0:
+		return
+	if index < 0 or index >= total:
+		return
+	if index == _last_focus_index:
+		return
+
+	_navigation_locked = true
+
+	var button: Button = quest_list.get_child(index)
+	if not is_instance_valid(button):
+		_navigation_locked = false
+		return
+
+	var quest: Quest = button.get_meta("quest") if button.has_meta("quest") else null
+
+	var old_index := _last_focus_index
```

Comment:
This third variable `old_index` is not necessary if we update the index at the bottom of the function.